Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -172,7 +172,7 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
 
   useEffect(() => {
     const index = currentDialog.triggers.findIndex(({ type }) => type === SDKKinds.OnBeginDialog);
-    if (index >= 0 && !designPageLocation.selected) {
+    if (index >= 0 && !designPageLocation.selected && designPageLocation.projectId === projectId) {
```

Comment:
To keep `projectId` `dialogId` always consistence, I think we should remove them from `designPageLocation` state. let components get from shell state.

Reply:
Previous related issue https://github.com/microsoft/BotFramework-Composer/issues/3022

Reply:
yes, These may take some time, We can do it in R11